Makefile

```diff
@@ -0,0 +1,153 @@
+# Makefile to help automate tasks
+PY := bin/python
+PIP := bin/pip
+PYTEST := bin/py.test
+THEBLUES := lib/python*/site-packages/theblues.egg-link
+FLAKE8 := bin/flake8
+SPHINX := bin/sphinx-apidoc
```

Comment:
I'm confused--a newline between SPHINX and TOX definitions, or are you asking for an additional newline between the definitions and the help command?